include/zephyr/bluetooth/conn.h

```diff
@@ -342,8 +378,10 @@ struct bt_conn_info {
 		/** BR/EDR Connection specific Info. */
 		struct bt_conn_br_info br;
 	};
-
+	/** Connection state. */
 	enum bt_conn_state state;
+	/** Security specific info. */
+	struct bt_security_info security;
```

Comment:
Should this be guraded by `CONFIG_BT_SMP`?

Reply:
No, without CONFIG_BT_SMP this still shows valid values i.e. security level 1

Reply:
I'm inclined to agree. It's typically going to be stack allocated and would make it easier to perhaps the checks without the guard.